app/controllers/dradis/plugins/calculators/cvss/issues_controller.rb

```diff
@@ -4,6 +4,16 @@ class IssuesController < ::IssuesController
     before_action :set_cvss_vector, only: :edit
 
     def edit
+      @cvss_version =
+        if @issue.fields['CVSSv4.BaseVector']
+          '4.0'
+        elsif @issue.fields['CVSSv3.Vector']&.include?('CVSS:3.1')
+          '3.1'
+        elsif @issue.fields['CVSSv3.Vector']&.include?('CVSS:3.0')
+          '3.0'
+        else
+          '4.0'
+        end
```

Comment:
`case` perhaps?

Also this looks more like a before_action than the actual content of the #edit action. Should you extract this logic to a helper?